docs/react/navigation.md

```diff
@@ -643,7 +643,7 @@ const MyComponent: React.FC = () => {
 
 ## More Information
 
-For more info on routing in React using React Router, check out their docs at [https://reacttraining.com/react-router/web](https://reacttraining.com/react-router/web).
+For more info on routing in React using the React Router implementation that Ionic uses under the hood, check out their docs at [https://v5.reactrouter.com/web](https://v5.reactrouter.com/web).
```

Comment:
Changed the language a bit to hint at the fact that this is the documentation for React Router that Ionic uses, not necessarily the latest React Router documentation